Android/GreenCircle/app/src/main/java/com/greencircle/framework/views/fragments/company/CreateCompanyFragment.kt

```diff
@@ -86,6 +87,16 @@ class CreateCompanyFragment : Fragment() {
                 Log.d("CreateCompanyFragment", "Google login failed")
             }
         }
+        viewModel.createCompanyResult.observe(viewLifecycleOwner) { result ->
+            // Handle the result here
+            if (result != null) {
```

Comment:
Refactor this, I don't see the need for the else printing the error instead of having a different logic.